graphql-database-manager-dynamo/src/main/java/com/fleetpin/graphql/database/manager/dynamo/DynamoHistoryBackupItem.java

```diff
@@ -0,0 +1,60 @@
+package com.fleetpin.graphql.database.manager.dynamo;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fleetpin.graphql.database.manager.util.HistoryBackupItem;
+import java.nio.ByteBuffer;
+import java.util.Map;
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
+
+public class DynamoHistoryBackupItem extends DynamoBackupItem implements HistoryBackupItem {
+
+	private String organisationIdType;
+
+	private ByteBuffer idRevision;
```

Comment:
`byte[]` probably play nicer with jackson

Reply:
Changed to `byte[]`.